### ISSUE TITLE
Toggle input style missing size

### ISSUE THREAD

**Issue by J-Kallunki** [vrk-kpa/suomifi-ui-components#268]
Should base on this format:
```html
<label for="suomifi-checkbox-id" class=“suomifi-checkbox">
  <input type="checkbox" id="suomifi-checkbox-id" class=“suomifi-checkbox_input" />
  <svg class=“suomifi-checkbox_icon" />
  <span>Label for the checkbox</span>
</label>
```
```css
.suomifi-checkbox .suomifi-checkbox_input:checked + .suomifi-checkbox_icon {
}
.suomifi-checkbox .suomifi-checkbox_input:focus + .suomifi-checkbox_icon {
}
.suomifi-checkbox .suomifi-checkbox_input {
  position: absolute;
  // SVG-ikcon size
  width: 1em;
  height: 1em;
  opacity: 0;
}
```
and use idGenerator()

_Originally posted by @J-Kallunki in https://github.com/vrk-kpa/suomifi-ui-components/issues/154#issuecomment-539826042_
